packages/jellyfish-api-core/__tests__/category/masternode/getGov.test.ts

```diff
@@ -19,7 +19,7 @@ describe('Masternode', () => {
 
   it('should getGov LP_DAILY_DFI_REWARD', async () => {
     const gov = await client.masternode.getGov('LP_DAILY_DFI_REWARD')
-    expect(gov.LP_DAILY_DFI_REWARD instanceof BigNumber).toStrictEqual(true)
+    expect(gov[0].LP_DAILY_DFI_REWARD instanceof BigNumber).toStrictEqual(true)
```

Comment:
are we reverting `getgov` array changes?

Reply:
reverted